bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -0,0 +1,157 @@
+            reply.println("only the pull request author and [Reviewers](https://openjdk.java.net/bylaws#reviewer) are allowed to use the `jep` command.");
+            return;
+        }
+
+        var args = command.args().trim();
+        if (args.isEmpty() || args.isBlank()) {
+            showHelp(reply);
+            return;
+        }
+
+        var labelNames = pr.labelNames();
+        if ("unneeded".equals(args) || "uneeded".equals(args)) {
+            if (labelNames.contains(JEP_LABEL)) {
+                pr.removeLabel(JEP_LABEL);
+            }
+            reply.println(unneededMarker);
+            reply.println("determined that the [JEP](https://openjdk.java.net/jeps) request " +
+                    "is not needed for this pull request.");
+            return;
+        }
+
+        // Get the issue
+        var jbsIssueOpt = getJepIssue(args, bot, reply);
+        if (jbsIssueOpt.isEmpty()) {
+            reply.println("The JEP issue was not found. Please make sure you have entered it correctly.");
+            showHelp(reply);
+            return;
+        }
+        var jbsIssue = jbsIssueOpt.get();
+
+        // Verify whether the issue type is a JEP
+        var issueType = jbsIssue.properties().get("issuetype");
+        if (issueType == null || !"JEP".equals(issueType.asString())) {
+            reply.println("The issue `" + jbsIssue.id() + "` is not a JEP. Please make sure you have entered it correctly.");
+            showHelp(reply);
+            return;
+        }
+
+        // Get the issue status
+        var issueStatus = jbsIssue.properties().get("status").get("name").asString();
+        var resolution = jbsIssue.properties().get("resolution");
+        String resolutionName = "";
+        if (resolution != null && !resolution.isNull() &&
+                resolution.get("name") != null && !resolution.get("name").isNull()) {
+            resolutionName = resolution.get("name").asString();
+        }
+
+        // Set the marker and output the result
+        var jepNumber = jbsIssue.properties().get(JiraProject.JEP_NUMBER).asString();
+        reply.println(String.format(jepMarker, jepNumber, jbsIssue.id(), jbsIssue.title()));
+        if ("Targeted".equals(issueStatus) || "Integrated".equals(issueStatus) ||
+            "Completed".equals(issueStatus) || ("Closed".equals(issueStatus) && "Delivered".equals(resolutionName))) {
+            reply.println("the JEP for this pull request, [JEP-" + jepNumber + "](" + jbsIssue.webUrl() + "), has already been targeted.");
+            if (labelNames.contains(JEP_LABEL)) {
+                pr.removeLabel(JEP_LABEL);
+            }
+        } else if ("Draft".equals(issueStatus) || "Submitted".equals(issueStatus) || "Candidate".equals(issueStatus) ||
+                "Proposed to Target".equals(issueStatus) || "Proposed to Drop".equals(issueStatus) || "Closed".equals(issueStatus)) {
```

Comment:
I think this should either just be "else", or we need an else clause after this for catching any unknown states. I think I prefer the former.

Reply:
I fixed it by using the former style and added a comment.